server/src/db/migrations/20230504135943_addImageUrlColumn.cjs

```diff
@@ -7,7 +7,7 @@
  */
 exports.up = async (knex) => {
     return knex.schema.alterTable("websites", (table) => {
-        table.string("imageUrl").notNullable().defaultTo(null)
+        table.string("imageUrl").notNullable().defaultTo("")
```

Comment:
- i dont have problems with requiring there be an imageURL, but it definitely should not default to anything, otherwise you can get false positives. remove the `defaultTo` altogether